changelog/0.40.md

```diff
@@ -0,0 +1,75 @@
+# v0.40.0
+
+Released 2024-08-05
+<!-- -->
+> [!IMPORTANT]
+> **Platform Administrator Notice(s)**
+>
+> - New environment variable `CK8S_K8S_INSTALLER`  is required for running `ck8s init`. There is a new command for listing supported installers with `ck8s k8s-installers`. It is assumed that nftables are used instead of iptables (Ubuntu 22.04 or newer) when `CK8S_K8S_INSTALLER` is set to `capi` (cluster-api). New generic openstack `CK8S_PROVIDER` for situations when apps is deployed on a openstack cloud but not safespring, elastx or citycloud.
```

Comment:
I think we can put these as separate notices:
```suggestion
> - New environment variable `CK8S_K8S_INSTALLER`  is required for running `ck8s init`. There is a new command for listing supported installers with `ck8s k8s-installers`. It is assumed that nftables are used instead of iptables (Ubuntu 22.04 or newer) when `CK8S_K8S_INSTALLER` is set to `capi` (cluster-api).
> - New generic openstack `CK8S_PROVIDER` for situations when apps is deployed on a openstack cloud but not safespring, elastx or citycloud.
```